src/app/pages/charging-stations/list/charging-stations-list-table-data-source.ts

```diff
@@ -325,11 +325,12 @@ export class ChargingStationsListTableDataSource extends TableDataSource<Chargin
 
   public buildTableFiltersDef(): TableFilterDef[] {
     if (this.isOrganizationComponentActive) {
+      const siteFilter = new SiteTableFilter().getFilterDef();
       return [
         // new ChargingStationTableFilter().getFilterDef(),
         new IssuerFilter().getFilterDef(),
-        new SiteTableFilter().getFilterDef(),
-        new SiteAreaTableFilter().getFilterDef()
+        siteFilter,
+        new SiteAreaTableFilter(null, [siteFilter]).getFilterDef(),
```

Comment:
sideIDs should be removed from the SiteAreaTableFilter and replaced by the filter deps.

Reply:
Done